cli/src/commands/version.rs

```diff
@@ -73,6 +79,11 @@ Report bugs: <https://github.com/martinvonz/jj/issues>
         writeln!(ui.stdout())?;
         writeln!(ui.stdout(), "Target: {}", env!("JJ_CARGO_TARGET"))?;
         writeln!(ui.stdout(), "Commit: {}", git_commit)?;
+        writeln!(
+            ui.stdout(),
+            "Release: {}",
```

Comment:
I'm not sure what the point of this field is. If the numeric version doesn't have a commit suffix, it's a release build. If it does have one, it's not.

Also I originally thought it was referring to whether the binary was built in release mode. It might separately be useful to add a big all caps blinking banner when the binary is built in debug mode.

Reply:
I guess it's mostly just a thing in case users look at it, since they may not immediately know what build they're using (they may miss finer details like that.)

Also, I'd like the build to include the actual Cargo features, but that's probably trickier.